core-services/prow/02_config/_config.yaml

```diff
@@ -1100,6 +1100,17 @@ tide:
     - 'dco-signoff: no'
     orgs:
     - cri-o
+  - labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/blocked-paths
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    repos:
+    - redhat-operator-ecosystem/playground
```

Comment:
IMO let's include the `release` repo here too, so Tide can merge our work in that repo for us there.

Reply:
Added!